curriculum/challenges/_meta/learn-lambda-functions-by-building-an-expense-tracker/meta.json

```
@@ -0,0 +1,192 @@
+{
+  "name": "Learn Lambda Functions by Building an Expense Tracker",
+  "isUpcomingChange": false,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
+  "dashedName": "learn-lambda-functions-by-building-an-expense-tracker",
+  "order": 2,
```

```suggestion
  "order": 66,
```

Set this to something random for now - it'll prevent steps from conflicting in the tests.

I'll confirm the order in my CI PR after this lands. :3

---

Ok.